java/spring-boot/src/main/java/benchmark/springboot/BenchmarkApplication.java

```diff
@@ -0,0 +1,34 @@
+package benchmark.springboot;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@SpringBootApplication
+public class BenchmarkApplication {
+
+  public static void main(String[] args) {
+    SpringApplication.run(BenchmarkApplication.class, args);
+  }
+
+  @RestController
+  public class BenchmarkController {
+
+    @GetMapping("/")
+    public void root() {
+    }
+
+    @GetMapping("/user/{id}")
+    public String userId(@PathVariable String id) {
```

Comment:
Could `{id}` be constrained ? (to be a numerical value)

Reply:
Sure it can but the instructions of the API to provide do not specify this. Moreover, what should be the response if the constraint does not pass?

Reply:
any http error 404, 400 => better is to follow java like guidelines

PS : This is to simplify work to do on https://github.com/the-benchmarker/web-frameworks/pull/1207